conductr_cli/sandbox_run_jvm.py

```diff
@@ -0,0 +1,255 @@
+from conductr_cli import sandbox_stop
+from conductr_cli.resolvers import bintray_resolver
+from conductr_cli.resolvers.bintray_resolver import BINTRAY_DOWNLOAD_REALM
+
+
+def run(args, features):
+    """
+    Starts the ConductR core and agent.
+
+    :param args: args parsed from the input arguments
+    :param features: list of features which are specified via -f switch.
+                     This is only relevant for Docker based sandbox since the features decides what port to expose
+    :return: a tuple containing list of core pids and list of agent pids
+    """
+    no_of_core_instances, no_of_agent_instances = instance_count(args.nr_of_containers)
+
+    validate_jvm_support()
+    validate_address_aliases(max(no_of_core_instances, no_of_agent_instances), args.interface, args.addr_range)
```

Comment:
Should the run command add the network interfaces in case they have not been specified instead of only validating them? By doing so the user would save one additional manual step which means we also would decrease the risk that the user misses this step / does not read the error message properly.